skycultures/bali/index.json

```diff
@@ -0,0 +1,859 @@
+{
+  "id": "bali",
+  "region": "South-eastern Asia",
+  "classification": ["Ethnographic"],
+  "fallback_to_international_names": false,
+  "constellations": [
+      {
+      "id": "CON bali 001",
+      "lines": [[
+        71795,
+        69673,
+        67927], [
+        69673,
+        72105,
+        73555], [
+        74666,
+        71075
+      ]],
+	  "image": {
+		  "file": "illustrations/Kala_Sungsang.png",
+		  "size": [900, 925],
+		  "anchors": [
+		  {"pos": [413, 137], "hip": 69673},
+		  {"pos": [291, 53], "hip": 67927},
+		  {"pos": [258, 661], "hip": 71075}]
+	  },
+      "common_name": {
+        "english": "upside-down demon",
```

Comment:
English names should be capitalized. The same I think for the `pronounce` fields.